modules/@apostrophecms/image-widget/index.js

```diff
@@ -9,6 +9,16 @@ module.exports = {
     placeholderClass: false,
     placeholderImage: 'jpg'
   },
+  widgetOperations: {
+    add: {
+      adjustImage: {
+        label: 'apostrophe:editImageAdjustments',
+        icon: 'image-edit-outline',
+        modal: 'AposImageRelationshipEditor',
```

Comment:
Does this makes the `relationshipEditor` options useless now?

Reply:
Used now